### DISCUSSION TITLE
[New Feature] Un-Hardcode TIBTRE0%d and related tags

### DISCUSSION THREAD

**Discussion by 4SGX** [Phobos-developers/Phobos#936] [Ideas and feedback]
### Description:
Currently ore spawners are hardcoded to be named TIBTRE01-TIBTRE09 and have the animation, speed, and palette hardcoded with them. I suggest unhardcoding the current ore generator tags so that new ones could be made with different settings. Related tags that seem to be hardcoded:
* IsAnimated
* SpawnsTiberium
* AnimationRate
* AnimationProbability

### Possible Implementation:
Requiring only SpawnsTiberium=yes to be a TIBTRE-like TerrainType.

**Comment by McPwny** [comment]
i mentioned these tags in the last post. i will add that while i was trying to make animated trees the other day, i did observe trees with the name "ATREE01-09" spawning ore and also that all 3 animation tags are ignored if spawnstiberium=no
